server/scripts/createSuperUser.ts

```diff
@@ -0,0 +1,42 @@
+import { PrismaClient } from '@prisma/client'
+import axios from 'axios'
+
+const prismaClient = new PrismaClient()
+
+const {
+  SUPERUSER_EMAIL,
+  SUPERUSER_PASSWORD,
+  IDENTITY_PROVIDER_URL,
+} = process.env
+
+const main = async () => {
+  if (!SUPERUSER_PASSWORD) throw Error('superuser password is required')
+  const user = {
+    name: 'admin',
+    password: SUPERUSER_PASSWORD as string,
+    email: SUPERUSER_EMAIL || 'admin@arkhn.com',
+  }
+  let form = new FormData()
+  form.append('name', user.name)
+  form.append('password', user.password)
+  form.append('email', user.email)
+  const res = await axios.post(`${IDENTITY_PROVIDER_URL}/signup`, form)
+  if (res.status === 500)
```

Comment:
Not sure what's the best but wouldn't checking if `res.status !== 200` be "safer"?